dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/RegistryDirectory.java

```diff
@@ -187,37 +177,38 @@ public void destroy() {
 
     @Override
     public synchronized void notify(List<URL> urls) {

```

Comment:
`public synchronized void notify(final List<URL> urls)`

Reply:
I guess no need to add final since this method is simple enough.